grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -88,6 +94,16 @@
 
     private static final boolean USE_COROUTINE_CONTEXT_INTERCEPTOR;
 
+    private static Map<String, HttpService> applyGrpcServiceToDecorators(
+            Map<String, List<DecoratorAndOrder>> mp,
+            GrpcService grpcService) {
+        final Map<String, HttpService> result = new HashMap<>();
+        for (Map.Entry<String, List<DecoratorAndOrder>> entry : mp.entrySet()) {
+            result.put(entry.getKey(), DecoratorUtil.applyDecorators(entry.getValue(), grpcService));
+        }
+        return result;
```

Comment:
Use immutable Map and stream?
```suggestion
        return mp.entrySet()
                 .stream()
                 .collect(toImmutableMap(Map.Entry::getKey,
                                         e -> DecoratorUtil.applyDecorators(e.getValue(), grpcService)));
```